### ISSUE TITLE
conditional typo?

### ISSUE THREAD

**Issue by abaird** [alisterscott/webdriver-user-agent#30]
https://github.com/alisterscott/webdriver-user-agent/blob/master/lib/webdriver-user-agent/driver.rb#L22

this currently reads
`if driver.browser == :firefox or :chrome`

did you mean:
`if driver.browser == :firefox or driver.browser == :chrome`?

**Comment by samnissen** [comment]
I believe there is no difference. Ruby seems to assume parenthesis around the content after the `==` so that `driver.browser == :firefox or :chrome` is the equivalent of `driver.browser == (:firefox or :chrome)`. Which is the same as `driver.browser == :firefox or driver.browser == :chrome`.

```ruby
$ irb
:001 > var = :foo
 => :foo
:002 > puts "works" if var == :foo or :bar
works
```

Tested on Ruby 2.1.2p95

If we are going to change this, I prefer `||` to `or` for [in case someone ever modifies the line to do assignment in-line](http://stackoverflow.com/questions/2083112/difference-between-or-and-in-ruby).

**Comment by abaird** [comment]
I don't think they are the same:
```
[1] pry(#<RSpec::Core::Example>)> a = :chrome
=> :chrome
[4] pry(#<RSpec::Core::Example>)> a == :firefox or :chrome
=> :chrome
[5] pry(#<RSpec::Core::Example>)> a == :firefox or a == :chrome
=> true
[6] pry(#<RSpec::Core::Example>)> a = :safari
=> :safari
[7] pry(#<RSpec::Core::Example>)> a == :firefox or :chrome
=> :chrome
[8] pry(#<RSpec::Core::Example>)> a == :firefox or a == :chrome
=> false
```
Since `:chrome` is a truthy value, then the behavior is the same using both methods. Probably not a big deal because Chrome and Firefox are the only browsers supported. However, if the result is to always execute the code in [here](https://github.com/alisterscott/webdriver-user-agent/blob/master/lib/webdriver-user-agent/driver.rb#L23-L25), why have the conditional at all? Maybe a better question is - under what circumstances does that code need to be run? If the answer is always, then you can probably remove the conditional.

**Comment by samnissen** [comment]
_tl;dr: The conditional is both wrong and unnecessary. I recommend a pull request that removes the conditional._

You are correct: This conditional doesn't function reasonably. You are also correct that the conditional doesn't seem to have value - nothing about the code contained in the conditional needs to be protected from a situation where the browser type is not one of those two values. Further, because of [this case statement](https://github.com/alisterscott/webdriver-user-agent/blob/master/lib/webdriver-user-agent/browser_options.rb#L36), I don't believe its even possible for the value to be anything other than the two values that would trigger the contained code's execution. I hope you will help fix this and that the fix will be approved.
